### PR TITLE
igxMonthPicker: fix months view KB navigation

### LINKED ISSUE #4190: igxMonthPicker - arrowdown and arrow up not working correctly inside months view
## Description  
**ArrowDown** and **ArrowUp** navigation does not work correctly inside months view

 * igniteui-angular version: 7.2.x
 * browser: N/A

## Steps to reproduce  

1. Instantiate a month picker in separate up
2. Use arrowup/arrowdown inside months view

## Result  
Does not navigate to the expected item.

## Expected result  
Should navigate to the expected item.

### LINKED ISSUE #4224: [igx-date-picker] unnecessary suffix "日" to the date part of the calendar.
## Description  
I'm trying to localize igx-date-picker to Japanese locale. I found that the formatting of date part is not what Japanese people expect. Until v7.1.9, there is no problem.

 * igniteui-angular version: 7.2.0-rc.3
 * browser: N/A

## Steps to reproduce  

1. Go to https://stackblitz.com/edit/igx-date-picker-template-issue
2. Show the dialog mode datepicker.
3. Show the dropdown mode datepicker.

## Result  
"日" is suffixed to each date number.
image

## Expected result  
"日" should be removed. `igx-date-picker` should format date parts as before.
image

## Attachments  
https://stackblitz.com/edit/igx-date-picker-template-issue

### PR DESCRIPTION
**Pull request by SAndreeva**
Closes #4190 
Closes #4224

### Additional information (check all that apply):
 - [x] Bug fix
 - [ ] New functionality
 - [ ] Documentation
 - [ ] Demos
 - [ ] CI/CD

### Checklist:
 - [x] All relevant tags have been applied to this PR
 - [ ] This PR includes unit tests covering all the new code
 - [ ] This PR includes API docs for newly added methods/properties
 - [ ] This PR includes `feature/README.MD` updates for the feature docs
 - [ ] This PR includes general feature table updates in the root `README.MD`
 - [ ] This PR includes `CHANGELOG.MD` updates for newly added functionality
 - [ ] This PR contains breaking changes
 - [ ] This PR includes `ng update` migrations for the breaking changes
 - [ ] This PR includes behavioral changes and the feature specification has been updated with them

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -7,7 +7,7 @@ All notable changes for each version of this project will be documented in this
     - **Feature** advanced keyboard navigation support has been added. Read up more information in the [ReadMe](https://github.com/IgniteUI/igniteui-angular/tree/master/projects/igniteui-angular/src/lib/calendar/README.md)
 
 - **New component** `IgxMonthPicker`:
-    - Provides the ability to pick a specific month. Read up more information in the [ReadMe](https://github.com/IgniteUI/igniteui-angular/tree/master/projects/igniteui-angular/src/lib/month-picker/README.md)
+    - Provides the ability to pick a specific month. Read up more information in the [ReadMe](https://github.com/IgniteUI/igniteui-angular/tree/master/projects/igniteui-angular/src/lib/calendar/month-picker/README.md)
```
Comment (bkulov) [inline CHANGELOG.md]: Month picker is a separate component similar to the date picker. Is there a reason why it's located under calendar folder?

File: projects/igniteui-angular/src/lib/calendar/months-view/months-view.component.ts [hunk 2]
```diff
@@ -259,11 +259,13 @@ export class IgxMonthsViewComponent implements ControlValueAccessor {
         }
 
         const months = this.dates.toArray();
-        if (months.indexOf(node) - 3 >= 0) {
-            const month = months[months.indexOf(node) - 3];
-
-            month.nativeElement.focus();
-         }
+        for (let index = months.indexOf(node) - 1; index >= 0; index--) {
+            if (node.nativeElement.getBoundingClientRect().top !== months[index].nativeElement.getBoundingClientRect().top &&
```
Comment (bkulov) [inline projects/igniteui-angular/src/lib/calendar/months-view/months-view.component.ts]: `node.nativeElement.getBoundingClientRect()` could be extracted as a variable outside of the cycle as it doesn't depend on the `index`.

File: projects/igniteui-angular/src/lib/calendar/months-view/months-view.component.ts [hunk 3]
```diff
@@ -280,11 +282,13 @@ export class IgxMonthsViewComponent implements ControlValueAccessor {
         }
 
         const months = this.dates.toArray();
-        if (months.indexOf(node) + 3 < months.length) {
-            const month = months[months.indexOf(node) + 3];
-
-            month.nativeElement.focus();
-         }
+        for (let index = months.indexOf(node) + 1; index < months.length; index++) {
+            if (node.nativeElement.getBoundingClientRect().top !== months[index].nativeElement.getBoundingClientRect().top &&
```
Comment (bkulov) [inline projects/igniteui-angular/src/lib/calendar/months-view/months-view.component.ts]: Same as previous comment.

File: CHANGELOG.md (hunk 1 above)
Comment (SAndreeva) [reply CHANGELOG.md]: It shares most of the code.

**Pull request merged**